.github/workflows/go.yml

```diff
@@ -24,6 +24,7 @@ jobs:
         - init-pod-mutator
         - pod-mutator
         - peering-request-webhook-init
+        - dispatcher
```

Comment:
Am I wrong or we came out with a different name for this component?

Reply:
How should we name it?

Reply:
@palexster Did you take notes? ServiceReplicator or something like that?

Reply:
Last time we came out with "CRDReplicator" but I am not sure if it will manage only CRDs or also other type of resources. So I kept the old name.

Reply:
That's the reason I used "service" here. Let's wait for Alex.

Reply:
IMHO Service has a pretty clear meaning in Kubernetes and can be misleading referring to service in this context. AFAIK In the near future, this component will handle CRDs replication across clusters, so CRDReplicator looks fine to me.